spec/services/create_project_activity_spec.rb

```diff
@@ -2,9 +2,10 @@
 
 RSpec.describe CreateProjectActivity do
   let(:beis) { create(:beis_organisation) }
-  let(:delivery_partner_organisation) { create(:delivery_partner_organisation) }
+  let(:delivery_partner_organisation) { create(:delivery_partner_organisation, organisation_type: organisation_type) }
   let(:user) { create(:administrator, organisation: delivery_partner_organisation) }
   let(:programme) { create(:programme_activity, organisation: beis) }
+  let(:organisation_type) { 10 }
```

Comment:
Do we need to `let` this? I can't see it used again? Could it be set explicitily? There are already loads of let here…

````
create(:delivery_partner_organisation, organisation_type:  10)
````

Reply:
No I think this is left over from a refactor!